### ISSUE TITLE
a8n/web: Changesets and ChangesetPlans are visually hard to distinguish

### ISSUE THREAD

**Issue by mrnugget** [sourcegraph/sourcegraph#7852]
When creating a Campaign from a CampaignPlan or when publishing a drafted Campaign, it's visually hard to follow what's happening. One of the reasons for that is that Changesets (created on codehost) and ChangesetPlans (not created on codehost yet, only diff in Sourcegraph database) are really hard to distinguish visually. Compare:

ChangesetPlans:

<img width="1139" alt="Screen Shot 2020-01-16 at 14 32 23" src="https://user-images.githubusercontent.com/1185253/72607897-c7fa6000-3921-11ea-9194-d5e625162f6c.png">

Changesets:

<img width="1136" alt="Screen Shot 2020-01-16 at 14 32 47" src="https://user-images.githubusercontent.com/1185253/72607872-bd3fcb00-3921-11ea-945d-2722d95fb08a.png">

I think we need a way to make it clearer to the user what _is_ and _is not_ on the codehost yet.

Maybe the solution to this involves making the overall campaign status ("preview", "draft", "created/published"/"closed") more visible on the whole page?

**Comment by eseliger** [comment]
I think looking at the recent improvements made to the UI, they look quite different nowadays

**Comment by mrnugget** [comment]
Yeah, the ticket here is kinda invalid now after #8008. We can still tweak it though :)
